frontend/src/v5/ui/controls/dueDate/dueDate.component.tsx

```diff
@@ -44,14 +46,20 @@ export const DueDate = ({ value: initialValue, disabled, onBlur }: DueDateProps)
 			<DatePicker
 				value={value}
 				open={open}
+				// onChange is a required prop in DatePicker, however it is not needed as onAccept works better
+				// (onChange triggers when changing year, onAccept only when a date is finally chosen)
 				onChange={() => true}
 				onAccept={onDateChange}
 				onClose={handleClose}
-				dayOfWeekFormatter={(day) => day[0].toUpperCase() + day[1]}
+				dayOfWeekFormatter={formatDayOfWeek}
 				disableHighlightToday
 				renderInput={({ ref, inputRef, ...props }) => (
 					<div ref={inputRef}>
-						<DueDateLabel onClick={onClickDueDate} clickable={!disabled} {...props} value={value} />
+						{ value ? (
+							<FilledDueDateLabel onClick={onClickDueDate} disabled={!disabled} {...props} value={value} />
+						) : (
+							<EmptyDueDateLabel onClick={onClickDueDate} disabled={!disabled} {...props} />
```

Comment:
When you pass `{...props}` to `*DueDateLabel`, you are overriding the `disabled` property. You can move the `{...props}` before `disabled` and pass the latter as is (i.e. `disabled={disabled}`)